metrics/ci_integrations/lib/client/buildkite/models/buildkite_token.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:ci_integration/client/buildkite/mappers/buildkite_token_scope_mapper.dart';
+import 'package:ci_integration/client/buildkite/models/buildkite_token_scope.dart';
+import 'package:equatable/equatable.dart';
+
+/// A class that represents a Buildkite access token needed for authorization.
+class BuildkiteToken extends Equatable {
+  /// A [List] of [BuildkiteTokenScope]s of this token.
+  final List<BuildkiteTokenScope> scopes;
+
+  @override
+  List<Object> get props => [scopes];
+
+  /// Creates an instance of the [BuildkiteToken] with the given [scopes].
+  const BuildkiteToken({
+    this.scopes,
+  });
+
+  /// Creates a new instance of the [BuildkiteToken] from the decoded JSON
+  /// object.
+  ///
+  /// Returns `null` if the given [json] is `null`.
+  factory BuildkiteToken.fromJson(Map<String, dynamic> json) {
+    if (json == null) return null;
+
+    const permissionMapper = BuildkiteTokenScopeMapper();
+    final permissionsList =
+        (json['scopes'] as List<dynamic>)?.map((element) => '$element');
```

Comment:
```suggestion
    final scopesValue = json['scopes'] as List<dynamic>;
    final scopesList = scopesValue?.map((element) => '$element');
```
Would it be more readable?